server/passport.js

```diff
@@ -69,6 +69,7 @@ module.exports = {
             function (req, userHandle, password, done) {
```

Comment:
this whole function has changed since what I see here. I think you should pull dev into your branch and reconcile any merges before requesting this pull.

Reply:
(I dont think there will be conflicts, I just dont think it was pulled first)